package.json

```diff
@@ -2,14 +2,16 @@
   "name": "@toznysecure/account-sdk",
   "version": "1.0.6",
   "description": "Javascript SDK for performing Tozny platform account level operations",
-  "main": "src/index.js",
+  "main": "dist/index.js",
```

Comment:
this points the whole project to the compiled code. when someone installs with `--production` flag, they don't need to download ts or build it locally on their machine. doing this requires that we commit the compiled code (and remember to build before release).

an alternative to this would be to require ts be installed with the package & run `build` as an install script.

Reply:
Suggest adding this or something similar to README per my note below